src/java/main/org/apache/zookeeper/server/ServerCnxnFactory.java

```diff
@@ -54,6 +54,7 @@
      */
     static final ByteBuffer closeConn = ByteBuffer.allocate(0);
 
+    private static String loginUser = System.getProperty("user.name", "<NA>");
```

Comment:
Not sure I understand what this represents

Reply:
loginUser is the user who started zk server.

Reply:
Perhaps this can be renamed? Maybe "serverUser" or something to clarify this is not the same thing as the user "logged in" on the client. Or at least a comment.

Reply:
changed to serverUser